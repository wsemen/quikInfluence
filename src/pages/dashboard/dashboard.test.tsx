import { render } from '@testing-library/react'
import DashBoard from '.'

it('renders DashBoard Page', () => {
  const { container } = render(<DashBoard />)
  expect(container).toMatchSnapshot()
});
