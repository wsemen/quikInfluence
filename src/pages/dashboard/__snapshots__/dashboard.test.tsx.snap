// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DashBoard Page 1`] = `
<div>
  <div
    class="chakra-stack css-n21gh5"
  >
    <nav
      class="css-njfis8"
    >
      <div
        class="css-1c714fi"
      >
        <img
          alt="quik-influence logo"
          class="chakra-image css-1ko1n4n"
        />
        <h1
          class="chakra-heading css-8mmjuy"
        >
          <div
            class="css-k008qs"
          >
            <p
              class="chakra-text css-67ii6d"
            >
              Quik
            </p>
            <p
              class="chakra-text css-0"
            >
              Influence
            </p>
          </div>
        </h1>
      </div>
      <div
        class="css-dsyim5"
      >
        <svg
          class="chakra-icon css-onkibi"
          focusable="false"
          viewBox="0 0 24 24"
        >
          <path
            d="M 3 5 A 1.0001 1.0001 0 1 0 3 7 L 21 7 A 1.0001 1.0001 0 1 0 21 5 L 3 5 z M 3 11 A 1.0001 1.0001 0 1 0 3 13 L 21 13 A 1.0001 1.0001 0 1 0 21 11 L 3 11 z M 3 17 A 1.0001 1.0001 0 1 0 3 19 L 21 19 A 1.0001 1.0001 0 1 0 21 17 L 3 17 z"
            fill="currentColor"
          />
        </svg>
      </div>
      <div
        class="css-0"
      >
        <script>
          (function setScript(initialValue) {
  var mql = window.matchMedia("(prefers-color-scheme: dark)");
  var systemPreference = mql.matches ? "dark" : "light";
  var persistedPreference = systemPreference;

  try {
    persistedPreference = localStorage.getItem("chakra-ui-color-mode");
  } catch (error) {
    console.log("Chakra UI: localStorage is not available. Color mode persistence might not work as expected");
  }

  var colorMode;

  if (persistedPreference) {
    colorMode = persistedPreference;
  } else if (initialValue === "system") {
    colorMode = systemPreference;
  } else {
    colorMode = initialValue != null ? initialValue : systemPreference;
  }

  if (colorMode) {
    /**
     * Keep in sync with \`root.set() {@file ./color-mode.utils.ts}
     */
    document.documentElement.style.setProperty("--chakra-ui-color-mode", colorMode);
    document.documentElement.setAttribute("data-theme", colorMode);
  }
})('light')
        </script>
      </div>
      <div
        class="css-iqqio3"
      >
        <button
          class="chakra-button css-158uetl"
          type="button"
        >
          Create account
        </button>
        <label
          class="chakra-switch css-tm8ajn"
        >
          <input
            class="chakra-switch__input"
            style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; width: 1px; margin: -1px; padding: 0px; overflow: hidden; white-space: nowrap; position: absolute;"
            type="checkbox"
            value=""
          />
          <span
            aria-hidden="true"
            class="chakra-switch__track css-14qxnv8"
          >
            <span
              class="chakra-switch__thumb css-0"
            />
          </span>
        </label>
        <span
          class="css-1ehzxp"
        >
          <span
            class="chakra-avatar css-o11zrc"
          >
            <div
              aria-label="Segun Adebayo"
              class="chakra-avatar__initials css-0"
              role="img"
            >
              SA
            </div>
          </span>
          <span
            class="css-8uhtka"
          >
            Segun
          </span>
        </span>
      </div>
    </nav>
    <div
      class="css-ww7h6m"
    >
      <div
        class="css-1rzqydi"
      >
        <div
          class="css-6ap22d"
        >
          <a
            class="chakra-link css-1e6bsnr"
            href="/dashboard"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-grip-horizontal fa-w-14 "
              data-icon="grip-horizontal"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 448 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M96 288H32c-17.67 0-32 14.33-32 32v64c0 17.67 14.33 32 32 32h64c17.67 0 32-14.33 32-32v-64c0-17.67-14.33-32-32-32zm160 0h-64c-17.67 0-32 14.33-32 32v64c0 17.67 14.33 32 32 32h64c17.67 0 32-14.33 32-32v-64c0-17.67-14.33-32-32-32zm160 0h-64c-17.67 0-32 14.33-32 32v64c0 17.67 14.33 32 32 32h64c17.67 0 32-14.33 32-32v-64c0-17.67-14.33-32-32-32zM96 96H32c-17.67 0-32 14.33-32 32v64c0 17.67 14.33 32 32 32h64c17.67 0 32-14.33 32-32v-64c0-17.67-14.33-32-32-32zm160 0h-64c-17.67 0-32 14.33-32 32v64c0 17.67 14.33 32 32 32h64c17.67 0 32-14.33 32-32v-64c0-17.67-14.33-32-32-32zm160 0h-64c-17.67 0-32 14.33-32 32v64c0 17.67 14.33 32 32 32h64c17.67 0 32-14.33 32-32v-64c0-17.67-14.33-32-32-32z"
                fill="currentColor"
              />
            </svg>
             
            Overview
          </a>
        </div>
      </div>
      <hr
        aria-orientation="vertical"
        class="chakra-divider css-78xnth"
      />
      <div
        class="css-1cwxi0e"
      >
        <div
          class="css-0"
        >
          <div
            class="css-ymykhv"
          >
            <div
              class="css-d6k11m"
            >
              <div
                class="css-vrmnwf"
              >
                <div
                  class="css-jqpl9f"
                />
                <p
                  class="chakra-text css-15gds93"
                >
                  Microphone Company
                </p>
              </div>
              <hr
                aria-orientation="horizontal"
                class="chakra-divider css-1upb9tn"
              />
              <div
                class="css-k3fo5g"
              >
                <div
                  class="css-e2q3qv"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-user fa-w-14 fa-sm "
                    data-icon="user"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style="margin: auto;"
                    viewBox="0 0 448 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
                      fill="currentColor"
                    />
                  </svg>
                  <p
                    class="chakra-text css-5yrxju"
                  >
                    29
                     Users
                  </p>
                </div>
                <div
                  class="css-e2q3qv"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-user fa-w-14 fa-sm "
                    data-icon="user"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style="margin: auto;"
                    viewBox="0 0 448 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
                      fill="currentColor"
                    />
                  </svg>
                  <p
                    class="chakra-text css-5yrxju"
                  >
                    223
                     Leads
                  </p>
                </div>
                <div
                  class="css-e2q3qv"
                >
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-money-bill-wave-alt fa-w-20 fa-sm "
                    data-icon="money-bill-wave-alt"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style="margin: auto;"
                    viewBox="0 0 640 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M621.16 54.46C582.37 38.19 543.55 32 504.75 32c-123.17-.01-246.33 62.34-369.5 62.34-30.89 0-61.76-3.92-92.65-13.72-3.47-1.1-6.95-1.62-10.35-1.62C15.04 79 0 92.32 0 110.81v317.26c0 12.63 7.23 24.6 18.84 29.46C57.63 473.81 96.45 480 135.25 480c123.17 0 246.34-62.35 369.51-62.35 30.89 0 61.76 3.92 92.65 13.72 3.47 1.1 6.95 1.62 10.35 1.62 17.21 0 32.25-13.32 32.25-31.81V83.93c-.01-12.64-7.24-24.6-18.85-29.47zM320 352c-44.19 0-80-42.99-80-96 0-53.02 35.82-96 80-96s80 42.98 80 96c0 53.03-35.83 96-80 96z"
                      fill="currentColor"
                    />
                  </svg>
                  <p
                    class="chakra-text css-5yrxju"
                  >
                    90
                    K
                  </p>
                </div>
              </div>
            </div>
            <br />
            <button
              class="chakra-button css-g81mmu"
              type="button"
            >
              Send Contact 
            </button>
            <br />
            <br />
            <button
              class="chakra-button css-1i4yx9k"
              type="button"
            >
              Send Contact 
            </button>
            <br />
            <br />
            <button
              class="chakra-button css-1830mhv"
              type="button"
            >
              Send Contact 
            </button>
            <br />
            <br />
            <div
              class="chakra-stack css-17r4hs2"
            >
              <h2
                class="chakra-heading css-0"
              >
                Filter
              </h2>
              <div
                class="chakra-accordion css-odz94x"
              >
                <div
                  class="chakra-accordion__item css-cn9ye4"
                >
                  <h2>
                    <button
                      aria-controls="accordion-panel-2"
                      aria-expanded="false"
                      class="chakra-accordion__button css-12ttzao"
                      data-index="0"
                      id="accordion-button-2"
                      type="button"
                    >
                      <div
                        class="css-8tcdg7"
                      >
                        Recent Activity
                      </div>
                      <svg
                        class="chakra-icon css-1hiix9"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M0,12a1.5,1.5,0,0,0,1.5,1.5h8.75a.25.25,0,0,1,.25.25V22.5a1.5,1.5,0,0,0,3,0V13.75a.25.25,0,0,1,.25-.25H22.5a1.5,1.5,0,0,0,0-3H13.75a.25.25,0,0,1-.25-.25V1.5a1.5,1.5,0,0,0-3,0v8.75a.25.25,0,0,1-.25.25H1.5A1.5,1.5,0,0,0,0,12Z"
                          fill="currentColor"
                        />
                      </svg>
                    </button>
                  </h2>
                  <div
                    class="chakra-collapse"
                    style="overflow: hidden; display: none; opacity: 0; height: 0px;"
                  >
                    <div
                      aria-labelledby="accordion-button-2"
                      class="chakra-accordion__panel css-4rbilf"
                      id="accordion-panel-2"
                      role="region"
                    >
                      <div
                        class="chakra-stack css-1qwdsol"
                      >
                        <span
                          class="css-1ehzxp"
                        >
                          Email Open
                        </span>
                        <span
                          class="css-1ehzxp"
                        >
                          Sms Open
                        </span>
                        <span
                          class="css-1ehzxp"
                        >
                          Non tag
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="chakra-accordion__item css-cn9ye4"
                >
                  <h2>
                    <button
                      aria-controls="accordion-panel-4"
                      aria-expanded="false"
                      class="chakra-accordion__button css-12ttzao"
                      data-index="1"
                      id="accordion-button-4"
                      type="button"
                    >
                      <div
                        class="css-8tcdg7"
                      >
                        status
                      </div>
                      <svg
                        class="chakra-icon css-1hiix9"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M0,12a1.5,1.5,0,0,0,1.5,1.5h8.75a.25.25,0,0,1,.25.25V22.5a1.5,1.5,0,0,0,3,0V13.75a.25.25,0,0,1,.25-.25H22.5a1.5,1.5,0,0,0,0-3H13.75a.25.25,0,0,1-.25-.25V1.5a1.5,1.5,0,0,0-3,0v8.75a.25.25,0,0,1-.25.25H1.5A1.5,1.5,0,0,0,0,12Z"
                          fill="currentColor"
                        />
                      </svg>
                    </button>
                  </h2>
                  <div
                    class="chakra-collapse"
                    style="overflow: hidden; display: none; opacity: 0; height: 0px;"
                  >
                    <div
                      aria-labelledby="accordion-button-4"
                      class="chakra-accordion__panel css-4rbilf"
                      id="accordion-panel-4"
                      role="region"
                    >
                      <div
                        class="chakra-form-control css-0"
                        role="group"
                      >
                        <div
                          class="chakra-select__wrapper css-42b2qy"
                        >
                          <select
                            class="chakra-select css-l65hh7"
                            data-test-id="select-component"
                            id=""
                          >
                            <option
                              value=""
                            >
                              Select ---
                            </option>
                            <option
                              data-testid="select-option"
                              value="active"
                            >
                              Active
                            </option>
                            <option
                              data-testid="select-option"
                              value="inactive"
                            >
                              Inactive
                            </option>
                            <option
                              data-testid="select-option"
                              value="incomplete"
                            >
                              Incomplete
                            </option>
                            <option
                              data-testid="select-option"
                              value="complete"
                            >
                              Complete
                            </option>
                          </select>
                          <div
                            class="chakra-select__icon-wrapper css-162mkon"
                          >
                            <svg
                              aria-hidden="true"
                              class="chakra-select__icon"
                              focusable="false"
                              role="presentation"
                              style="width: 1em; height: 1em; color: currentColor;"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                                fill="currentColor"
                              />
                            </svg>
                          </div>
                        </div>
                      </div>
                      <div
                        class="css-bkneeg"
                        spacing="4"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="chakra-accordion__item css-cn9ye4"
                >
                  <h2>
                    <button
                      aria-controls="accordion-panel-6"
                      aria-expanded="false"
                      class="chakra-accordion__button css-12ttzao"
                      data-index="2"
                      id="accordion-button-6"
                      type="button"
                    >
                      <div
                        class="css-8tcdg7"
                      >
                        age
                      </div>
                      <svg
                        class="chakra-icon css-1hiix9"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M0,12a1.5,1.5,0,0,0,1.5,1.5h8.75a.25.25,0,0,1,.25.25V22.5a1.5,1.5,0,0,0,3,0V13.75a.25.25,0,0,1,.25-.25H22.5a1.5,1.5,0,0,0,0-3H13.75a.25.25,0,0,1-.25-.25V1.5a1.5,1.5,0,0,0-3,0v8.75a.25.25,0,0,1-.25.25H1.5A1.5,1.5,0,0,0,0,12Z"
                          fill="currentColor"
                        />
                      </svg>
                    </button>
                  </h2>
                  <div
                    class="chakra-collapse"
                    style="overflow: hidden; display: none; opacity: 0; height: 0px;"
                  >
                    <div
                      aria-labelledby="accordion-button-6"
                      class="chakra-accordion__panel css-4rbilf"
                      id="accordion-panel-6"
                      role="region"
                    >
                      <div
                        class="chakra-form-control css-0"
                        role="group"
                      >
                        <div
                          class="chakra-select__wrapper css-42b2qy"
                        >
                          <select
                            class="chakra-select css-l65hh7"
                            data-test-id="select-component"
                            id=""
                          >
                            <option
                              value=""
                            >
                              Select ---
                            </option>
                            <option
                              data-testid="select-option"
                              value="18-25"
                            >
                              18-25
                            </option>
                            <option
                              data-testid="select-option"
                              value="26-32"
                            >
                              26-32
                            </option>
                            <option
                              data-testid="select-option"
                              value="33-50"
                            >
                              33-50
                            </option>
                            <option
                              data-testid="select-option"
                              value="51-above"
                            >
                              51-above
                            </option>
                          </select>
                          <div
                            class="chakra-select__icon-wrapper css-162mkon"
                          >
                            <svg
                              aria-hidden="true"
                              class="chakra-select__icon"
                              focusable="false"
                              role="presentation"
                              style="width: 1em; height: 1em; color: currentColor;"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                                fill="currentColor"
                              />
                            </svg>
                          </div>
                        </div>
                      </div>
                      <div
                        class="css-bkneeg"
                        spacing="4"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="chakra-accordion__item css-cn9ye4"
                >
                  <h2>
                    <button
                      aria-controls="accordion-panel-8"
                      aria-expanded="false"
                      class="chakra-accordion__button css-12ttzao"
                      data-index="3"
                      id="accordion-button-8"
                      type="button"
                    >
                      <div
                        class="css-8tcdg7"
                      >
                        gender
                      </div>
                      <svg
                        class="chakra-icon css-1hiix9"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M0,12a1.5,1.5,0,0,0,1.5,1.5h8.75a.25.25,0,0,1,.25.25V22.5a1.5,1.5,0,0,0,3,0V13.75a.25.25,0,0,1,.25-.25H22.5a1.5,1.5,0,0,0,0-3H13.75a.25.25,0,0,1-.25-.25V1.5a1.5,1.5,0,0,0-3,0v8.75a.25.25,0,0,1-.25.25H1.5A1.5,1.5,0,0,0,0,12Z"
                          fill="currentColor"
                        />
                      </svg>
                    </button>
                  </h2>
                  <div
                    class="chakra-collapse"
                    style="overflow: hidden; display: none; opacity: 0; height: 0px;"
                  >
                    <div
                      aria-labelledby="accordion-button-8"
                      class="chakra-accordion__panel css-4rbilf"
                      id="accordion-panel-8"
                      role="region"
                    >
                      <div
                        class="chakra-form-control css-0"
                        role="group"
                      >
                        <div
                          class="chakra-select__wrapper css-42b2qy"
                        >
                          <select
                            class="chakra-select css-l65hh7"
                            data-test-id="select-component"
                            id=""
                          >
                            <option
                              value=""
                            >
                              Select ---
                            </option>
                            <option
                              data-testid="select-option"
                              value="male"
                            >
                              Male
                            </option>
                            <option
                              data-testid="select-option"
                              value="female"
                            >
                              Female
                            </option>
                            <option
                              data-testid="select-option"
                              value="other"
                            >
                              Other
                            </option>
                          </select>
                          <div
                            class="chakra-select__icon-wrapper css-162mkon"
                          >
                            <svg
                              aria-hidden="true"
                              class="chakra-select__icon"
                              focusable="false"
                              role="presentation"
                              style="width: 1em; height: 1em; color: currentColor;"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                                fill="currentColor"
                              />
                            </svg>
                          </div>
                        </div>
                      </div>
                      <div
                        class="css-bkneeg"
                        spacing="4"
                      />
                    </div>
                  </div>
                </div>
                <div
                  class="chakra-accordion__item css-cn9ye4"
                >
                  <h2>
                    <button
                      aria-controls="accordion-panel-10"
                      aria-expanded="false"
                      class="chakra-accordion__button css-12ttzao"
                      data-index="4"
                      id="accordion-button-10"
                      type="button"
                    >
                      <div
                        class="css-8tcdg7"
                      >
                        application
                      </div>
                      <svg
                        class="chakra-icon css-1hiix9"
                        focusable="false"
                        viewBox="0 0 24 24"
                      >
                        <path
                          d="M0,12a1.5,1.5,0,0,0,1.5,1.5h8.75a.25.25,0,0,1,.25.25V22.5a1.5,1.5,0,0,0,3,0V13.75a.25.25,0,0,1,.25-.25H22.5a1.5,1.5,0,0,0,0-3H13.75a.25.25,0,0,1-.25-.25V1.5a1.5,1.5,0,0,0-3,0v8.75a.25.25,0,0,1-.25.25H1.5A1.5,1.5,0,0,0,0,12Z"
                          fill="currentColor"
                        />
                      </svg>
                    </button>
                  </h2>
                  <div
                    class="chakra-collapse"
                    style="overflow: hidden; display: none; opacity: 0; height: 0px;"
                  >
                    <div
                      aria-labelledby="accordion-button-10"
                      class="chakra-accordion__panel css-4rbilf"
                      id="accordion-panel-10"
                      role="region"
                    >
                      <div
                        class="chakra-form-control css-0"
                        role="group"
                      >
                        <div
                          class="chakra-select__wrapper css-42b2qy"
                        >
                          <select
                            class="chakra-select css-l65hh7"
                            data-test-id="select-component"
                            id=""
                          >
                            <option
                              value=""
                            >
                              Select ---
                            </option>
                            <option
                              data-testid="select-option"
                              value="quikSession"
                            >
                              QuikSession
                            </option>
                            <option
                              data-testid="select-option"
                              value="idemandBeauti"
                            >
                              IdemandBeauti
                            </option>
                            <option
                              data-testid="select-option"
                              value="asd_exchange"
                            >
                              ASD Exchange
                            </option>
                          </select>
                          <div
                            class="chakra-select__icon-wrapper css-162mkon"
                          >
                            <svg
                              aria-hidden="true"
                              class="chakra-select__icon"
                              focusable="false"
                              role="presentation"
                              style="width: 1em; height: 1em; color: currentColor;"
                              viewBox="0 0 24 24"
                            >
                              <path
                                d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                                fill="currentColor"
                              />
                            </svg>
                          </div>
                        </div>
                      </div>
                      <div
                        class="css-bkneeg"
                        spacing="4"
                      />
                    </div>
                  </div>
                </div>
              </div>
              <button
                class="chakra-button css-1i4yx9k"
                type="button"
              >
                Filter Search
              </button>
            </div>
            <br />
            <br />
            <br />
            <div
              class="chakra-form-control css-0"
              role="group"
            >
              <input
                class="chakra-input css-1lohn59"
                id="field-11"
                placeholder="Email"
                type="email"
                value=""
              />
            </div>
            <br />
            <div
              class="chakra-form-control css-0"
              role="group"
            >
              <label
                class="chakra-form__label css-1lxtfqi"
                data-testid="textInput-label"
                for=""
                id="field-12-label"
              >
                POC
              </label>
              <input
                class="chakra-input css-1lohn59"
                id="field-12"
                placeholder="Email"
                type="email"
                value=""
              />
            </div>
            <br />
            <div
              class="chakra-form-control css-0"
              role="group"
            >
              <label
                class="chakra-form__label css-1lxtfqi"
                data-invalid=""
                data-testid="textInput-label"
                for=""
                id="field-13-label"
              >
                POC
              </label>
              <input
                aria-describedby="field-13-feedback"
                aria-invalid="true"
                class="chakra-input css-1lohn59"
                id="field-13"
                placeholder="Email"
                type="email"
                value=""
              />
              <div
                aria-live="polite"
                class="chakra-form__error-message css-s4fyc4"
                data-testid="textInput-error"
                id="field-13-feedback"
              >
                Invalid email
              </div>
            </div>
            <br />
            <div
              class="chakra-form-control css-0"
              role="group"
            >
              <label
                class="chakra-form__label css-160ag6o"
                for=""
                id="field-14-label"
              >
                Payout Model
              </label>
              <div
                class="chakra-select__wrapper css-42b2qy"
              >
                <select
                  class="chakra-select css-l65hh7"
                  data-test-id="select-component"
                  id=""
                >
                  <option
                    value=""
                  >
                    Select Payout Model
                  </option>
                  <option
                    data-testid="select-option"
                    value="test"
                  >
                    test
                  </option>
                  <option
                    data-testid="select-option"
                    value="atest"
                  >
                    atest
                  </option>
                  <option
                    data-testid="select-option"
                    value="btest"
                  >
                    btest
                  </option>
                </select>
                <div
                  class="chakra-select__icon-wrapper css-162mkon"
                >
                  <svg
                    aria-hidden="true"
                    class="chakra-select__icon"
                    focusable="false"
                    role="presentation"
                    style="width: 1em; height: 1em; color: currentColor;"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
              </div>
            </div>
            <br />
            <div
              class="chakra-form-control css-0"
              role="group"
            >
              <div
                class="chakra-select__wrapper css-42b2qy"
              >
                <select
                  class="chakra-select css-js7600"
                  data-test-id="select-component"
                  id=""
                >
                  <option
                    value=""
                  >
                    Select ---
                  </option>
                  <option
                    data-testid="select-option"
                    value="test"
                  >
                    test
                  </option>
                  <option
                    data-testid="select-option"
                    value="atest"
                  >
                    atest
                  </option>
                  <option
                    data-testid="select-option"
                    value="btest"
                  >
                    btest
                  </option>
                </select>
                <div
                  class="chakra-select__icon-wrapper css-162mkon"
                >
                  <svg
                    aria-hidden="true"
                    class="chakra-select__icon"
                    focusable="false"
                    role="presentation"
                    style="width: 1em; height: 1em; color: currentColor;"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
              </div>
            </div>
            <br />
            <div
              class="chakra-form-control css-0"
              role="group"
            >
              <div
                class="chakra-select__wrapper css-42b2qy"
              >
                <select
                  aria-describedby="field-16-feedback"
                  aria-invalid="true"
                  class="chakra-select css-js7600"
                  data-test-id="select-component"
                  id=""
                >
                  <option
                    value=""
                  >
                    Select ---
                  </option>
                  <option
                    data-testid="select-option"
                    value="test"
                  >
                    test
                  </option>
                  <option
                    data-testid="select-option"
                    value="atest"
                  >
                    atest
                  </option>
                  <option
                    data-testid="select-option"
                    value="btest"
                  >
                    btest
                  </option>
                </select>
                <div
                  class="chakra-select__icon-wrapper css-162mkon"
                >
                  <svg
                    aria-hidden="true"
                    class="chakra-select__icon"
                    focusable="false"
                    role="presentation"
                    style="width: 1em; height: 1em; color: currentColor;"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
              </div>
              <div
                aria-live="polite"
                class="chakra-form__error-message css-s4fyc4"
                id="field-16-feedback"
              >
                select one to proceed
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
